programmers/로또의-최고-순위와-최저-순위.js

```diff
@@ -0,0 +1,22 @@
+function solution(lottos, win_nums) {
+    let answer = [];
+    let zeroCnt = 0;
+    let Cnt = 0;
+    let rank = [6,6,5,4,3,2,1];
+    
+    let numSet = new Set();
```

Comment:
1. 배열 선언할 때 , <- 이후에 띄어쓰기로 구분해주기
2. set 객체와 배열은 const 변수 타입 해주기
3. 그리고 Cnt 변수명은 cnt가 나아보임

```suggestion
    const rank = [6, 6, 5, 4, 3, 2, 1];
    const numSet = new Set();
    const answer = [ ];
    
    let zeroCnt = 0;
    let cnt = 0;
```